docs/get-started/developers/index.md

```diff
@@ -26,32 +89,178 @@ Read this section to get started and e-mail [design-system@redhat.com](mailto:de
+    </svg>
+    <h3 slot="header">Design tokens</h3>
+    <p>Design tokens are how we translate our design language decisions into code.</p>
+    <rh-cta slot="footer">
+      <a href="tokens">Our design tokens</a>
+    </rh-cta>
+  </rh-card>
+  <rh-card>
+    <svg slot="header" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36">
+      <path d="M31 11.38H18.26l-6.82-6.82a.62.62 0 0 0-1.06.44v6.38H5a.62.62 0 0 0-.62.62v19a.62.62 0 0 0 .62.62h26a.62.62 0 0 0 .62-.62V12a.62.62 0 0 0-.62-.62Zm-.62 13H25a.62.62 0 0 0-.62.62v5.38h-5.76V12.62h11.76Zm-.89 1.24-3.87 3.87v-3.87ZM11.62 6.51l5.76 5.75v17.23l-5.76-5.75Zm-6 6.11h4.76V24a.59.59 0 0 0 .18.44l5.93 5.94H5.62Zm20.89 17.76 3.87-3.87v3.87Z"/>
+    </svg>
+    <h3 slot="header">Documentation</h3>
+    <p>This website offers guidance about how to use our 
+      <a href="/elements">elements</a> and <a href="/patterns">patterns</a>.
+      Learn how to apply them accessibily with
+      <a href="/accessibility/development/">developer-specific guidelines</a>.</p>
+    <rh-cta slot="footer">
+      <a href="/elements">Our elements</a>
+    </rh-cta>
+  </rh-card>
+  <rh-card>
+    <svg slot="header" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36">
+      <path d="M31 4.38H5a.62.62 0 0 0-.62.62v26a.62.62 0 0 0 .62.62h26a.62.62 0 0 0 .62-.62V5a.62.62 0 0 0-.62-.62Zm-.62 26H5.62V5.62h24.76Z"/>
+      <path d="M17.24 21.54a.62.62 0 0 0 .74-.49l1.24-5.84a.64.64 0 0 0-.48-.75.63.63 0 0 0-.74.54l-1.25 5.84a.64.64 0 0 0 .49.7ZM23.12 18 21 20.11a.63.63 0 1 0 .88.89l2.56-2.56a.62.62 0 0 0 0-.89L21.88 15a.63.63 0 1 0-.88.88Zm-9 3a.63.63 0 0 0 .88-.89L12.88 18 15 15.87c.59-.56-.31-1.46-.88-.88l-2.56 2.56a.62.62 0 0 0 0 .89Z"/>
+    </svg>
+    <h3 slot="header">GitHub repositories</h3>
+    <p>Explore our code, roadmaps, and discussions in the 
+      <a href="https://github.com/RedHat-UX/red-hat-design-system">Red Hat Design System repo</a> and the 
+      <a href="https://github.com/RedHat-UX/red-hat-design-tokens">Red Hat Design Tokens repo</a>.</p>
+    <rh-cta slot="footer">
+      <a href="https://github.com/RedHat-UX/">View our repos</a>
+    </rh-cta>
+  </rh-card>
+</div>
+
+
+## About Web Components
+
+### What are Web Components?
+
+Every web component is a custom HTML element. Web Components are made with standard web platform APIs, and not only with JavaScript frameworks. Those standards include [Custom Elements][ce]; [Shadow DOM][sd]; [HTML templates][te]; [CSS Custom Properties][cssprop] and [Shadow Parts][csspart]; [Element Internals][internals]; [ECMAScript Modules][esm]; and others. These combine to enable developers to write reusable and encapsulated UI elements which work with the browser's in-built component model.
```

Comment:
Probably better to use an unordered list for:

> Those standards include:
>  * Custom Elements
>  * Shadow Dom...

Reply:
"in-built" or "built-in"? I'd vote for built-in. YMMV.

Reply:
I think the list will help readability. Good idea.